src/server/migrations/20200317214852_add_users_schema.js

```diff
@@ -0,0 +1,29 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('users', function(table) {
+    table
+      .increments('id')
+      .notNullable()
+      .primary();
```

Comment:
Please add another field here ("uid" string [null])
Please follow the updated DB tables changes.

Reply:
The uid should not be null. Firebase assigns a uid for each user.

Reply:
Yes, uid should not be null but according to Daniel will keep it null now, may be later it will be changed.

Reply:
We will be hard coding the user data as per Daniel.

Reply:
@Daniel please clear this as yesterday we updated new DB as per your suggestion.